src/main/resources/application.yaml

```diff
@@ -1,7 +1,3 @@
 server:
   # TODO: change to your application's port number
   port: 4550
-
-management:
```

Comment:
that's annoying, it means you can't change log levels on the fly without getting credentials for it and you won't get the extended health information without more changes :(

Reply:
sounds like u want it enabled during dev? surely not once it's in the wild?

Reply:
that would be enough yeah